tests/unit/model.py

```diff
@@ -97,7 +97,8 @@ def sample_obj(self):
 class TestMetadata(ModelTest):
 
     def sample_obj(self):
-        node = Node(label='node-99',
+        from haas.ext.obm.ipmi import Ipmi
+        return Node(label='node-99',
```

Comment:
These tests are just a basic "if I create one of these, does it explode?" Skipping the line below shouldn't cause a test failure, but you are effectively skipping the test.